proto/viam/app/v1/billing.proto

```diff
@@ -105,10 +106,61 @@ message GetItemizedInvoiceResponse {
   Invoice invoice = 1;
 }
 
+// Begin Suggestion 1:
+message GetPaymentMethodCardRequest {
+  string org_id = 1;
+}
+
+message GetPaymentMethodCardResponse {
+  PaymentMethodCard method = 1;
+}
+// End Suggestion 1
+
+
+// Begin Suggestion 2:
+message GetPaymentMethodRequest {
+  string org_id = 1;
+}
+
+// would go at top of file with the rest of the
+// non-request/response message types
+enum PaymentMethodType {
+  PAYMENT_METHOD_TYPE_UNSPECIFIED = 0;
+  PAYMENT_METHOD_TYPE_CARD = 1;
+}
+
+message GetPaymentMethodResponse {
+  PaymentMethodType type = 1;
+  // defined if type is PAYMENT_METHOD_TYPE_CARD
+  optional PaymentMethodCard method = 2;
+}
+// End Suggestion 2
+
+
+message GetBalanceSummaryRequest {
+  string org_id = 1;
+}
+message GetBalanceSummaryResponse {
```

Comment:
The original balance field terms are also slightly outdated, and because they don't map exactly to the FE they are slightly confusing i would probably change it to something like:
statement_balance -> oustanding balance 
current_balance -> current_month_balance 

The usage breakdowns also would be good here.

Reply:
We also need the total_cost_with_discount, total_cost_without_discount, and discount here too.

Reply:
I have restructured this to be include previous invoices and to focus primarily on the top-banner and the invoice list. I have kept that month-based info in `GetCurrentMonthUsage`. How do you feel about this approach?